extension/src/popup/views/ReviewAuth/index.tsx

```diff
@@ -195,16 +200,25 @@ const AuthDetail = ({
 }) => {
   const { t } = useTranslation();
   const details = getInvocationDetails(authEntry.rootInvocation());
+  const invocations = details.filter(
+    (detail) => detail.type === "invoke",
+  ) as FnArgsInvoke[];
+  const createWasms = details.filter(
+    (detail) => detail.type === "wasm",
+  ) as FnArgsCreateWasm[];
+  const createSacs = details.filter(
+    (detail) => detail.type === "sac",
```

Comment:
future improvement: might be nice to store these arg types in an enum for easier reuse

Reply:
makes sense, I think we can refactor this to possibly just drop all of this in favor of some tweaks to `buildInvocationTree`